test/models/user_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class UserTest < ActiveSupport::TestCase
```

Comment:
this class can be deleted for the moment

Reply:
Class deleted 
![removeTest](https://user-images.githubusercontent.com/58991676/71357993-2ff99880-2566-11ea-93b6-0eff4f0c1a0e.png)